src/triggersubscriberhandler.ts

```diff
@@ -14,7 +14,7 @@ interface EmailData {
     email: string,
     listId: string,
     emailGroup: string,
-    triggeredSendKey: string,
+    triggeredSendKey?: string,
```

Comment:
Oh yes. Actually I am probably going to remove `monapt`.
And probably typescript. (Maybe, ish).